packages/api/portfolioDrafts/funding/createFundingStep.test.ts

```diff
@@ -408,47 +337,249 @@ describe("All Clins in Funding Step input validation tests", () => {
-      validationMessage: ValidationMessage.INVALID_FUNDING_AMOUNT,
-      invalidParameterName: "total_clin_value",
-      invalidParameterValue: "-1",
-    });
-    expect(errors).toContainEqual({
-      clinNumber: "0001",
-      validationMessage: ValidationMessage.INVALID_FUNDING_AMOUNT,
-      invalidParameterName: "obligated_funds",
-      invalidParameterValue: "-1",
-    });
   });
 });
 
-describe("Error response creation tests", () => {
+describe("Error response creation tests", function () {
   it("should return error response that includes error_map in response body", () => {
+    jest.spyOn(console, "error").mockImplementation(() => jest.fn()); // suppress output
     const obj = { errors: { propertyA: "property_value", propertyB: "property_value" } };
     const invalidProperties: Record<string, unknown> = obj;
-    const response = createValidationErrorResponse(invalidProperties);
-    expect(response).toBeInstanceOf(ValidationErrorResponse);
-    expect(JSON.parse(response.body).message).toMatch(/Invalid input/);
-    expect(JSON.parse(response.body).error_map).toEqual(obj);
-  });
-  it("should return error response containing all invalid properties in error_map", async () => {
-    const result = await handler(validRequestBadData);
-    expect(result).toBeInstanceOf(ValidationErrorResponse);
-    expect(JSON.parse(result.body).message).toMatch(/Invalid input/);
+    expect(() => {
+      createBusinessRulesValidationErrorResponse(invalidProperties);
+    }).toThrow(Error("Business rules validation failed"));
   });
   it("should throw error if invalid properties input is empty", () => {
     const emptyInvalidProperties: Record<string, unknown> = {};
     expect(() => {
-      createValidationErrorResponse(emptyInvalidProperties);
+      createBusinessRulesValidationErrorResponse(emptyInvalidProperties);
     }).toThrow(Error("Parameter 'invalidProperties' must not be empty"));
   });
   it("should throw error if invalid properties input has empty string as key", () => {
     const emptyStringKeyInvalidProperties: Record<string, unknown> = {
       "": { propertyA: "property_value", propertyB: "property_value" },
     };
     expect(() => {
-      createValidationErrorResponse(emptyStringKeyInvalidProperties);
+      createBusinessRulesValidationErrorResponse(emptyStringKeyInvalidProperties);
     }).toThrow(Error("Parameter 'invalidProperties' must not have empty string as key"));
   });
 });
+
+/**
+ * Sample Funding Step with valid data
+ * @returns a complete FundingStep with good data that should not cause input validation errors
+ */
+function mockFundingStepGoodData(): FundingStep {
+  const mockTaskOrderFile: FileMetadataSummary = {
+    id: "b91db32f-40fa-4225-9885-b032f0d229fe",
+    name: "TO_12345678910.pdf",
+  };
+  return {
+    task_orders: [
+      {
+        task_order_number: "12345678910123",
+        task_order_file: mockTaskOrderFile,
+        clins: [mockClin()],
+      },
+    ],
+  };
+}
```

Comment:
Everything here and below can probably go in `commonFundingMockData.ts` and then import them.

Reply:
Huh I think I may have accidently overwritten that file after merging, I'll put everything in that file.

Reply:
This test needs several adjustments wrt to recently merged #375